scripts/qa_crowbarsetup.sh

```diff
@@ -5366,6 +5381,7 @@ function onadmin_qa_test
 # $cct_skip_func_tests -> optional, functional tests will be skipped if value is 1, default is 0
 function onadmin_run_cct
 {
+    pre_hook $FUNCNAME
```

Comment:
that is why we also call it in the very last line in the 'main' part of qa_crowbarsetup